task_templates/pipelines/r_lang_anomaly_detection/create_pipeline.R

```diff
@@ -0,0 +1,11 @@
+create_pipeline<-function(X) {
```

Comment:
We should do an estimator instead of a pipeline, I thought we wanted to steer away from pipelines?

Reply:
@jmbannon I thought we were just going to put the pipelines folder under estimators? I'm fine with either, do you have a strong opinion?

Reply:
right but we want to encourage users to use DR preprocessing and not have to write single-task BPs w/everything in a pipeline. I think we should make this an estimator. Thoughts @MikhailYakubovskiy ?

Reply:
right, we want to have a few pipelines as they are useful in some use cases (e.g. once we have PPS, they'll likely require pipelines?). But generally we want users to rely on blueprints. We also want to have fewer templates to maintain. Let's have a simple R anomaly estimator, but not create a pipeline for it.

Reply:
Sold! I'll redo this as an estimator then